website_sale_product_attribute_value_filter_existing/tests/test_website_sale_product_attribute_value_filter_existing.py

```diff
@@ -23,6 +23,9 @@ def setUp(self):
         self.product_attribute_value_blue = ProductAttributeValue.create(
             {"name": "Test blue", "attribute_id": self.product_attribute.id}
         )
+        self.product_attribute_value_yelow = ProductAttributeValue.create(
+            {"name": "Test yelow", "attribute_id": self.product_attribute.id}
```

Comment:
```suggestion
            {"name": "Test yellow", "attribute_id": self.product_attribute.id}
```

Reply:
I was hungry and I ate the "L"! thanks :)